recipe/0001-Updated-protobuf-to-3.19.1.patch

```diff
@@ -0,0 +1,40 @@
+From 6522756e8bf5ded25f99343211a63e3c274a139e Mon Sep 17 00:00:00 2001
+From: Nishidha Panpaliya <npanpa23@in.ibm.com>
+Date: Mon, 28 Mar 2022 06:24:18 -0400
+Subject: [PATCH] Updated protobuf to 3.19.1
+
+---
+ WORKSPACE | 13 ++++---------
+ 1 file changed, 4 insertions(+), 9 deletions(-)
+
+diff --git a/WORKSPACE b/WORKSPACE
+index 62db091..2f29042 100644
+--- a/WORKSPACE
++++ b/WORKSPACE
+@@ -2,19 +2,14 @@ workspace(name = "tensorflow_metadata")
+ 
+ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+ 
+-http_archive(
+-    name = "bazel_skylib",
+-    sha256 = "2ef429f5d7ce7111263289644d233707dba35e39696377ebab8b0bc701f7818e",
+-    urls = ["https://github.com/bazelbuild/bazel-skylib/releases/download/0.8.0/bazel-skylib.0.8.0.tar.gz"],
+-)
+ 
+ # 310ba5ee72661c081129eb878c1bbcec936b20f0 is based on 3.8.0 with a fix for protobuf.bzl.
```

Comment:
I think this comment is not longer needed

Reply:
Removed. Thanks!